bundle/manifests/argocd-operator.clusterserviceversion.yaml

```diff
@@ -1102,7 +1102,7 @@ spec:
                   valueFrom:
                     fieldRef:
                       fieldPath: metadata.annotations['olm.targetNamespaces']
-                image: quay.io/argoprojlabs/argocd-operator:v0.7.0
+                image: quay.io/argoprojlabs/argocd-operator@sha256:5541a1c2323016b767f53bcadf696ebd199903d2048f7bd2aee377a332ea5f2c
```

Comment:
confused about this still, I'm not sure why it's inputting a SHA here when it should be `v0.7.0` :( (this was an automatic change and I think I remember this happening last release too)

Reply:
That is fine. CSV in the bundle will use the SHA instead of tag.